### ISSUE TITLE
Support single instanceTypes for spot instance

### ISSUE THREAD

**Issue by calclavia** [weaveworks/eksctl#1511]
**Why do you want this feature?**
It seems like AWS autoscaling groups support having just a single instance type for spot instance if I manually configure it. It would be great to support the ability to launch spot instances of a single type because there are specific use cases where that matters. For example, I have a use case where I need to spawn a GPU job with 8 GPUs, and no other instance type would work for this case. 

**What feature/behavior/change do you want?**

Ability to specify a single instance type for spot instances.
```
  - name: gpu-dev
    instanceType: mixed
    desiredCapacity: 0
    minSize: 0
    maxSize: 5
    volumeSize: 256
    volumeType: gp2
    instancesDistribution:
      onDemandPercentageAboveBaseCapacity: 0
      instanceTypes:
        - p2.xlarge
...
```

**Comment by sayboras** [comment]
it's already done in https://github.com/weaveworks/eksctl/pull/1772, and available from 0.14.0.

@calclavia @martina-if we can close this if I am not wrong :)

**Comment by calclavia** [comment]
Awesome thanks!
